bin/localizely_upload_main.dart

```diff
@@ -30,6 +30,11 @@ Future<void> main(List<String> arguments) async {
     'api-token',
     help: 'Localizely API token.',
   );
+  argParser.addOption(
+    'arb-dir',
+    help: 'Path of the arb files.',
+    defaultsTo: defaultArbDir,
```

Comment:
We should update handling of default value here also (similar to the download suggestion)